third_party/hamcrest-core.jar

```diff
@@ -1 +0,0 @@
-hamcrest-core-1.3.jar
```

Comment:
They are just symlinks. The build script uses the specific file and not the symlink. This was the main point of failure on Windows.